### DISCUSSION TITLE
Bulk add IP addresses

### DISCUSSION THREAD

**Discussion by edroszcz** [netbox-community/netbox#8628] [Q&A]
Hi,

I do feel a bit stupid regarding this one. But I hope someone can help me figure it out :)

I am trying to add a range of IP-addresses that are reserved for DHCP.

But I struggle with finding the correct syntax for bulk creating the IP addresses.

What would the bulk syntax for adding all IP addresses in the following range look like?

10.100.70.10-10.100.71.254

Regards Erik

**Comment by candlerb** [comment]
```
10.100.70.[10-254]/24
```

If you go to Add IP Address, Bulk Create tab, then you should see the following (at least in v3.1.7):

> Specify a numeric range to create multiple IPs.
> Example: 192.0.2.[1,5,100-254]/24
